app/app.go

```diff
@@ -311,6 +349,7 @@ type OsmosisApp struct {
 	TxFeesKeeper         *txfeeskeeper.Keeper
 	SuperfluidKeeper     superfluidkeeper.Keeper
 	GovKeeper            *govkeeper.Keeper
+	WasmKeeper           wasm.Keeper
```

Comment:
Would it be alright to make this a pointer, for consistency / easing potential hooks? (superfluid not being pointer receiver is a bug right now)

Reply:
We do this to reduce the error chance when passing in keepers to hooks or other module initializations. (We messed up once via missing a subtle change, and this is what made us have the emergency upgrade from v5 to v6)

Happy to do this in a second PR though

Reply:
Happy to change this